src/core/Meganav/MeganavPanel.tsx

```diff
@@ -0,0 +1,123 @@
+import FeaturedLink from "../FeaturedLink";
+import { ProductName, products } from "../ProductTile/data";
+import ProductTile from "../ProductTile";
+
+export const MeganavPanel = ({
+  displayProductTile,
+  panelLeft,
+  panelLeftClassName,
+  panelRightHeading,
+  panelRightItems,
+  panelRightBottom,
+}: {
+  displayProductTile?: boolean;
+  panelLeft?: FlyoutPanelHighlight;
+  panelLeftClassName?: string;
+  panelRightHeading?: string;
+  panelRightItems: FlyoutPanelList[];
+  panelRightBottom?: React.ReactNode;
+}) => {
+  return (
+    <>
+      <div className="flex flex-col md:flex-row gap-x-24 bg-neutral-000 dark:bg-neutral-1300">
+        <div
+          className={cn(
+            "flex-[7] flex-shrink-0 grid-cols-1 xs:grid-cols-2 group",
+            panelLeftClassName,
+          )}
+          onClick={
+            !displayProductTile
+              ? () => (window.location.href = panelLeft?.url as string)
+              : undefined
+          }
+        >
+          {displayProductTile
+            ? [
+                "pubsub",
+                "spaces",
+                "chat",
+                "liveSync",
+                "assetTracking",
+                "liveObjects",
+              ].map((product) => (
+                <ProductTile
+                  name={product as ProductName}
+                  key={product}
+                  selected={false}
+                  onClick={() =>
+                    (window.location.href = products[product as ProductName]
+                      ?.link as string)
+                  }
+                  animateIcons={true}
+                />
+              ))
+            : panelLeft && (
+                <>
+                  <div className="w-full p-24">
+                    <h4 className="ui-text-h4 text-neutral-1300 dark:text-neutral-000">
+                      {panelLeft.heading}
+                    </h4>
+                    <p className="ui-text-p3 text-neutral-800 dark:text-neutral-500 mt-8">
+                      {panelLeft.content}
+                    </p>
+                    <FeaturedLink
+                      url={panelLeft.url}
+                      additionalCSS="text-neutral-1300 dark:text-neutral-000 mt-16 ui-text-p3 hover:text-neutral-1300 dark:hover:text-neutral-000"
+                      iconColor="text-orange-600"
+                      groupHoverClassName="group-hover/meganav-panel:left-0"
+                    >
+                      {panelLeft.labelLink}
+                    </FeaturedLink>
+                  </div>
+                  <div className="flex justify-end">
+                    <img
+                      src={panelLeft.image}
+                      alt={panelLeft.heading}
+                      className="w-full z-10 rounded-lg"
+                    />
+                  </div>
+                </>
+              )}
+        </div>
+        <div className="mt-12 md:mt-0 flex-[3] flex-shrink-0 flex flex-col justify-between">
```

Comment:
On the mobile nav, there's a bit too big of a gap above the panels in my opinion - I would get rid of `mt-12` (and the class next to it). Looking at the design this looks like a good thing to do